Add JC filter plugin example

kellyjonbrazil opened the pull request:
##### SUMMARY
Added an example using the JC filter plugin.


##### ISSUE TYPE
- Docs Pull Request

+label: docsite_pr

##### SUMMARY
The JC filter plugin has been available in the community.general collection for some time now, so I thought it would be good to add an example to the existing documentation site.

##### ISSUE TYPE
- Docs Pull Request


##### COMPONENT NAME
community.general.jc

##### ADDITIONAL INFORMATION
This is a documentation addition. Let me know if there is a better place for this example. Thanks!

kellyjonbrazil committed:
Add JC filter plugin example

##### SUMMARY
Added an example using the JC filter plugin.


##### ISSUE TYPE
- Docs Pull Request

+label: docsite_pr

kellyjonbrazil committed:
Add JC to parsing the cli section and fix code block issue

kellyjonbrazil committed:
code block fixes

kellyjonbrazil committed:
Add blank line after code block

kellyjonbrazil commented:
+label: ready_for_review

Akasurde commented on docs/docsite/rst/network/user_guide/cli_parsing.rst:
```diff
@@ -56,6 +56,8 @@ The ``cli_parse`` module includes the following cli_parsing plugins:
 ``pyats``
   Uses the parsers included with the Cisco Test Automation & Validation Solution
+``jc``
+  A python module which converts the output of dozens of popular linux/unix/macOS/Windows commands and file-types to python dictionaries or lists of dictionaries. Note: this filter plugin can be found in the ``community.general`` collection.
```
```suggestion
  A python module that converts the output of dozens of popular Linux/UNIX/macOS/Windows commands and file types to python dictionaries or lists of dictionaries. Note: this filter plugin can be found in the ``community.general`` collection.
```

Akasurde commented on docs/docsite/rst/network/user_guide/cli_parsing.rst:
```diff
@@ -583,6 +585,29 @@ The task sets the follow fact as the ``interfaces`` fact for the host:
+Parsing with JC
+-----------------
+
+JC is a Python library that converts the output of dozens of common linux/unix/macOS/Windows command-line tools and file-types to python dictionaries or lists of dictionaries for easier parsing. JC is available as a filter plugin in the ``community.general`` collection.
```
```suggestion
JC is a Python library that converts the output of dozens of common Linux/UNIX/macOS/Windows command-line tools and file-types to python dictionaries or lists of dictionaries for easier parsing. JC is available as a filter plugin in the ``community.general`` collection.
```

Akasurde commented:
@kellyjonbrazil Thanks for submitting this PR. A few nit-picks, rest LGTM. Waiting for others to comment.

kellyjonbrazil committed:
Update docs/docsite/rst/network/user_guide/cli_parsing.rst

make suggested edits to a few words.

Co-authored-by: Abhijeet Kasurde <akasurde@redhat.com>

kellyjonbrazil committed:
Update docs/docsite/rst/network/user_guide/cli_parsing.rst

change capitalization

Co-authored-by: Abhijeet Kasurde <akasurde@redhat.com>

kellyjonbrazil committed:
Minor formatting

made 'Python' lower-case and removed hyphen in 'file-types' to be consistent with the summary section

Akasurde reviewed (approved):
LGTM

samccann commented:
Thanks @kellyjonbrazil for the Ansible fix!
